eng/tools/GenerateFiles/Directory.Build.targets.in

```diff
@@ -83,7 +83,7 @@
       <RuntimePackRuntimeIdentifiers Condition="'$(DotNetBuild)' == 'true'">%(RuntimePackRuntimeIdentifiers);$(TargetRuntimeIdentifier)</RuntimePackRuntimeIdentifiers>
     </KnownFrameworkReference>
 
-    <KnownAppHostPack Update="Microsoft.NETCore.App">
+    <KnownAppHostPack Update="Microsoft.NETCore.App" Condition=" '$(PackAsToolShimRuntimeIdentifiers)' == '' ">
```

Comment:
Consider adding a comment explaining why this is conditioned on PackAsToolShimRuntimeIdentifiers here.

Reply:
Will do - thanks!

Reply:
Fixed with https://github.com/dotnet/aspnetcore/pull/61282/commits/976f05c01e4808d8383a7d0b85dbcfedaa263617